src/Traits/GenericActionsTrait.php

```diff
@@ -54,9 +54,16 @@ abstract public function paginate($object = null, array $settings = []);
      */
     protected function loadFilteredChildren(string $id): array
     {
+        $folder = $this->Objects->loadObject($id, 'folders');
```

Comment:
Should we pass the `$folder` entity as method parameter instead of its `$id`?

Reply:
makes sense